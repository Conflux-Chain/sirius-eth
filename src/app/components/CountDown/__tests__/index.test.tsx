import React from 'react';
import { render } from '@testing-library/react';
import { CountDown } from '..';
import { MemoryRouter } from 'react-router';

describe('<CountDown  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <MemoryRouter>
        <CountDown from={1600690230000} to={1603282230000} />
      </MemoryRouter>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
