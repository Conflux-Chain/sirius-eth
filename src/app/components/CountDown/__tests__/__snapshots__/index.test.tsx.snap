// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CountDown  /> should match snapshot 1`] = `null`;
